transitfeed/util.py

```diff
@@ -226,9 +226,10 @@ def CheckVersion(problems, latest_version=None):
 
 
 def _MaxVersion(versions):
-  versions = filter(None, versions)
-  versions.sort(lambda x,y: -cmp([int(item) for item in x.split('.')],
-                                 [int(item) for item in y.split('.')]))
+  versions = [item for item in versions if item]
+  version_tuple = lambda x: tuple([int(item) for item in x.split('.')])
+  versions.sort(key=version_tuple, reverse=True)
```

Comment:
Instead of `sort`, do max, since we seem to only care about the maximal value. Something like:

    versions = [item for item in versions if item]
    if len(versions) == 0:
        return None
    version_tuple = lambda x: tuple(int(item) for item in x.split('.')) 
    return max(versions, key=version_tuple)

Reply:
thanks for good feedback!